site/content/docs/5.2/examples/form/index.html

```diff
@@ -0,0 +1,384 @@
+          <h1 class="display-1">Become a resseler</h1>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="container-xxl pt-4">
+    <div class="row">
+      <div class="col-md-8 mx-auto">
+        <p class="ll-sm h3 mb-4">You are a professional and wish to resell Orange products to your
+          customers?
+          <br>Please fill the form and we will get back to you shortly.
+        </p>
+        <form class="needs-validation mb-5" novalidate>
+          <div class="row">
+            <div class="col-12">
+              <p class="fw-bold mb-4">
+                <span class="text-primary">*</span> indicates required
+              </p>
+            </div>
+            <div class="col-md-4 col-6 mb-3">
+              <label for="selectTitle" id="selectTitleLabel" class="form-label is-required">Title</label>
+              <select class="form-select" id="selectTitle" aria-labelledby="selectTitleLabel selectTitleFeedback"
+                required>
+                <option selected disabled value="" aria-hidden="true">Select a title</option>
+                <option value="1">Miss</option>
+                <option value="2">Ms.</option>
+                <option value="3">Mrs.</option>
+              </select>
+              <div class="valid-feedback">
+                Looks good!
+              </div>
+              <div id="selectTitleFeedback" class="invalid-feedback">
+                Please select a valid title.
+              </div>
+            </div>
+            <div class="col-12">
+              <div class="mb-3">
+                <label for="inputFirstName" id="inputFirstNameLabel" class="form-label is-required">First name</label>
+                <input type="text" class="form-control" id="inputFirstName"
+                  aria-labelledby="inputFirstNameLabel inputFirstNameFeedback" required>
+                <div class="valid-feedback">
+                  Looks good!
+                </div>
+                <div id="inputFirstNameFeedback" class="invalid-feedback">
+                  Please enter a valid first name.
+                </div>
+              </div>
+              <div class="mb-3">
+                <label for="inputLastName" id="inputLastNameLabel" class="form-label is-required">Last name</label>
+                <input type="text" class="form-control" id="inputLastName"
+                  aria-labelledby="inputLastNameLabel inputLastNameFeedback" required>
+                <div class="valid-feedback">
+                  Looks good!
+                </div>
+                <div id="inputLastNameFeedback" class="invalid-feedback">
+                  Please enter a valid last name.
+                </div>
+              </div>
+              <div class="mb-3">
+                <label for="inputEmail" id="inputEmailLabel" class="form-label is-required">Email</label>
+                <input type="email" class="form-control" id="inputEmail" placeholder="name@example.com"
```

Comment:
There's no placeholder in the DSM. Do we keep it?

Reply:
We can remove it. But if we want to present a truly accessible form we need to provide a way to display the desired format.

Maybe in the error message like:

```
<div id="inputEmailFeedback" class="invalid-feedback">
  Please provide a valid email. Valid email address format: "name@example.com"
</div>
```

What's your opinion @Aniort?

Reply:
Honestly IMO nobody needs an information nowadays about the format of an email address in a form. Plus, "Valid email address format: "name@example.com"" is limitating since the specification allows several formats we aren't even aware of. Most of the time, backend doesn't even check the format of email addresses.

Reply:
I know. But it's mandatory for form accessibility: https://www.w3.org/TR/WCAG21/#error-suggestion
I will have a discussion about it with the accessibility experts.